README.md

```diff
@@ -236,6 +237,59 @@ an upload comment to be set. This can be achieved using the
 `--comment` option. If not set, Tailor will automatically use
 `Updated extension to <version>` as comment.
 
+### Create a local artefact of an extension
+
+You can generate a local artefact of your extension using the
+`create-artefact` command. This will generate a zip archive
+ready to be uploaded to TER (which is not covered by this
+command, have a look at the [`ter:publish`](#publish-a-new-version-of-an-extension-to-ter)
+command instead).
+
+Provide the version number and extension key as arguments
+followed by the path to the extension directory or an artefact
+(a zipped version of your extension). The latter can be either
+local or a remote file.
+
+Using `--path`:
+
+```bash
+./vendor/bin/tailor create-artefact 1.2.0 my_extension --path=/path/to/my_extension
+```
+
+Using a local `--artefact`:
+
+```bash
+./vendor/bin/tailor create-artefact 1.2.0 my_extension --artefact=/path/to/any-zip-file/my_extension.zip
+```
+
+Using a remote `--artefact`:
+
+```bash
+./vendor/bin/tailor create-artefact 1.2.0 my_extension --artefact=https://github.com/my-name/my_extension/archive/1.2.0.zip
+```
+
+Using the root directory:
+
+```bash
+./vendor/bin/tailor create-artefact 1.2.0 my_extension
+```
+
+If the extension key is defined as environment variable or
+in your `composer.json`, it can also be skipped. So using the
+current root directory the whole command simplifies to:
+
+```bash
+./vendor/bin/tailor create-artefact 1.2.0
+```
+
+**Important**: A couple of directories and files are excluded
```

Comment:
You're right. Duplicated content from `ter:publish` command was intentional because the same applies to the new `create-artefact` as well. Do you have an idea for improvement in this area?

Reply:
Could we add this information once at some general place in the doc and link to it from both places?

Reply:
I came up with an adjustment in 6f33c9c where I moved this section to a dedicated extra section. In addition, I converted all highlighted sections to native GitHub admonitions for better readability.